### PR TITLE
SIG-Arch meetings to biweekly, move KEP subproject to PM, remove todos

### PR DESCRIPTION
**Pull request by jdumars**
Reference:  https://youtu.be/78EKp6fAsqE

### CODE REVIEW

File: sig-architecture/charter.md [hunk 1]
```diff
@@ -31,6 +30,7 @@ The Architecture SIG maintains and evolves the design principles of Kubernetes,
 
 - KEPs that do not have architectural implications or impact are managed by their respective sponsoring SIG(s)
 - The release enhancement delivery [process] that is part of the SIG-Release Release Team [subproject]
+- The KEP process itself is now managed by SIG-PM and architecture subprjects are a stakeholder
```
Comment (smarterclayton) [inline sig-architecture/charter.md]: Typo

Comment (bgrant0607) [inline sig-architecture/charter.md]: typo: subprojects

**Comment by bgrant0607** [comment]
Otherwise LGTM, thanks

**Comment by jdumars** [comment]
Fixed.

**Comment by jdumars** [comment]
@justaugustus PTAL 
@timothysc PTAL & LGTM if it really LGTY TYVM! EIEIO

**Comment by justaugustus** [comment]
@jdumars -- I was just about to mention the squash, but I see you added the label. :)
/lgtm
/meow space

File: sig-architecture/charter.md [hunk 2]
```diff
@@ -51,12 +51,6 @@ and opts-in to updates and modifications to [sig-governance].
 
 Federation of Subprojects as defined in [sig-governance]
 
-### To do
```
Comment (quinton-hoole-2) [inline sig-architecture/charter.md]: Why are we removing these todo's when they have not yet been resolved?

Comment (jdumars) [reply sig-architecture/charter.md]: https://youtu.be/78EKp6fAsqE?t=1162

**Pull request merged**